sans/reduction.ipynb

```diff
@@ -177,6 +177,18 @@
     "    #                      PolynomialOrder=3, OutputUnfittedData=True)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "beamline = sc.neutron.TofBeamline(\n",
+    "    path=['source_position', 'sample_position', 'position'],\n",
+    "    scatter='sample_position')\n",
+    "elastic = sc.neutron.ElasticScattering(theta='theta');"
```

Comment:
Here we could add extra options (or use different converters) to support things such as gravity.